.template/variants/web/spec/support/disable_animation.rb

```diff
@@ -52,7 +52,7 @@ def inject(fragment)
   }
 </style>
       EOF
-      fragment.gsub(%r{</head>}, disable_animations + '</head>')
+      fragment.gsub(%r{</head>}, "#{disable_animations}</head>")
```

Comment:
this is fixed in develop branch, please rebase that @rafayetn :)

Reply:
Done!